packages/extension-chakra-storefront/src/components/with-react-intl/with-react-intl.tsx

```diff
@@ -92,7 +90,7 @@ const withReactIntl = <P extends object>(WrappedComponent: React.ComponentType<P
                 // NOTE: if you update this value, please also update the following npm scripts in `template-retail-react-app/package.json`:
                 // - "extract-default-translations"
                 // - "compile-translations:pseudo"
-                defaultLocale={DEFAULT_LOCALE}
+                defaultLocale={config.defaultLocale}
```

Comment:
I see now. From my earlier comment, I was wondering why this `defaultLocale` exists. 

I think let's rename it to something like `defaultAppLocale`. Because it's the default locale for the _app_. It's a different concept than the default locale found in ECOM sites like RefArch.